.circleci/config.yml

```diff
@@ -714,12 +714,16 @@ jobs:
         environment:
           # we need CGO because of go-sqlite3
           CGO_ENABLED: 1
+          # Reduce golangci-lint memory usage (default is 100)
+          GOGC: 20
     working_directory: /go/src/github.com/grafana/grafana
     steps:
       - checkout
       - run:
           name: Lint Go
-          command: "make lint-go"
+          command: |
+            make scripts/go/bin/golangci-lint
```

Comment:
Just to verify, after this is merged `vet` `revive` `revive-alerting` and `gosec` checks will no more executed. This is intentional right?

Reply:
@papagian Actually, no :/ Thanks for reminding me.

Reply:
golangci-lint keeps getting killed though, even after these tweaks :(